examples/powertools-examples-cloudformation/pom.xml

```diff
@@ -0,0 +1,123 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>powertools-examples-cloudformation</artifactId>
+    <packaging>jar</packaging>
+    <name>AWS Lambda Powertools for Java library Examples - CloudFormation</name>
+
+    <parent>
+        <artifactId>powertools-examples</artifactId>
+        <groupId>software.amazon.lambda</groupId>
+        <version>1.14.0</version>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-core</artifactId>
+            <version>1.2.2</version>
```

Comment:
Suggest dependency versions stored in either root pom of the repo, or examples parent if only relevant to examples. This continues the pattern with the other modules and avoids divergence between different pieces.

Reply:
Fixing this